examples/nodejs/aws/lambda-examples/advanced-compression-cdk/infrastructure/package-lock.json

```diff
@@ -0,0 +1,7236 @@
+{
+  "name": "advanced-compression-cdk",
+  "version": "0.1.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "advanced-compression-cdk",
+      "version": "0.1.0",
+      "dependencies": {
+        "aws-cdk-lib": "2.140.0",
+        "constructs": "^10.0.0",
+        "source-map-support": "^0.5.21"
+      },
+      "bin": {
+        "advanced-compression-cdk-2": "bin/advanced-compression-cdk.js"
+      },
+      "devDependencies": {
+        "@types/jest": "^29.5.12",
+        "@types/node": "20.12.7",
+        "@typescript-eslint/eslint-plugin": "5.59.11",
+        "aws-cdk": "2.140.0",
+        "eslint": "8.42.0",
+        "eslint-config-prettier": "8.8.0",
+        "eslint-plugin-import": "2.27.5",
+        "eslint-plugin-node": "11.1.0",
+        "eslint-plugin-prettier": "4.2.1",
+        "jest": "^29.7.0",
+        "prettier": "2.8.8",
+        "ts-jest": "^29.1.2",
+        "ts-node": "^10.9.2",
+        "typescript": "~5.4.5"
+      }
+    },
+    "node_modules/@ampproject/remapping": {
+      "version": "2.3.0",
+      "resolved": "https://momento-prod-401011790710.d.codeartifact.us-west-2.amazonaws.com/npm/npm-upstream/@ampproject/remapping/-/remapping-2.3.0.tgz",
```

Comment:
It looks like you were logged into our custom repository when this lock file was generated. You'll need to delete this and regenerate it after making sure that `.npmrc` is deleted from your home directory.